PsfRuntime/Config.cpp

```diff
@@ -255,11 +255,58 @@ void LogCountedStringW(const char* name, const wchar_t* value, std::size_t lengt
 
 static const psf::json_object* g_CurrentExeConfig = nullptr;
 
+// Note: Recursive
+FILE* find_json(std::filesystem::path basefolder)
+{
+    FILE* fRet = NULL;

```

Comment:
Can you move these declarations to right before they are being used, and if you can combine the deceleration and assignment?  I understand that FindFileData can be declareed outside of the do loop since it is assigned different values per iteration of the loop.

Reply:
Done. I assume that this is part of the coding standards still not documented for contributors?